Tests/XCUITests/HistoryTests.swift

```diff
@@ -103,215 +113,264 @@ class HistoryTests: BaseTestCase {
         XCTAssertEqual(app.alerts.buttons["OK"].exists, false)
     }
 
-    func testRecentlyClosedOptionAvailable() {
-        navigator.performAction(Action.CloseURLBarOpen)
+    func testRecentlyClosedNoWebsiteOpen() {
+        // "Recently Closed Tabs List" is empty by default
+        closeKeyboard()
         navigator.nowAt(NewTabScreen)
-        navigator.goto(HistoryRecentlyClosed)
-        waitForNoExistence(app.tables["Recently Closed Tabs List"])
+        navigator.goto(LibraryPanel_History)
+        waitForExistence(app.tables[HistoryPanelA11y.tableView])
+        XCTAssertFalse(app.tables.cells.staticTexts[bookOfMozilla["label"]!].exists)
+        XCTAssertTrue(app.tables[HistoryPanelA11y.tableView].staticTexts[emptyRecentlyClosedMesg].exists)
+    }
 
-        // Go to the default web site  and check whether the option is enabled
-        navigator.nowAt(LibraryPanel_History)
-        navigator.goto(HomePanelsScreen)
-        navigator.openURL(path(forTestPage: "test-mozilla-book.html"))
-        navigator.nowAt(BrowserTab)
+    func testRecentlyClosedWebsiteOpen() {
+        // Open "Book of Mozilla"
+        openBookOfMozilla()
+
+        // The tab, which is still opened, is not included in the "Recently Closed Tabs List"
         navigator.goto(BrowserTabMenu)
-        // Workaround to bug 1508368
-        navigator.goto(LibraryPanel_Bookmarks)
-        navigator.goto(HomePanelsScreen)
-        navigator.performAction(Action.OpenNewTabFromTabTray)
-        navigator.performAction(Action.CloseURLBarOpen)
-        navigator.nowAt(NewTabScreen)
         navigator.goto(LibraryPanel_History)
-        navigator.goto(HistoryRecentlyClosed)
-        waitForNoExistence(app.tables["Recently Closed Tabs List"])
-        navigator.nowAt(LibraryPanel_History)
-        navigator.goto(HomePanelsScreen)
+        waitForExistence(app.tables[HistoryPanelA11y.tableView])
+        XCTAssertFalse(app.tables.cells.staticTexts[bookOfMozilla["label"]!].exists)
+        XCTAssertTrue(app.tables[HistoryPanelA11y.tableView].staticTexts[emptyRecentlyClosedMesg].exists)
+    }
 
-        // Now go back to default website close it and check whether the option is enabled
-        navigator.openURL(path(forTestPage: "test-mozilla-book.html"))
-        waitUntilPageLoad()
-        waitForTabsButton()
-        navigator.goto(TabTray)
-        navigator.performAction(Action.AcceptRemovingAllTabs)
-        navigator.performAction(Action.CloseURLBarOpen)
+    func testRecentlyClosedWebsiteClosed() {
```

Comment:
This test is still failing for me.
Once you close the first tab, in line 141, the test goes to HomePanelScreen with the keyboard shown, you have to close it before being able to go to HistoryRecentlyClosed.
so you need these lines after L141:
```
waitForExistence(app.buttons["urlBar-cancel"], timeout: 15)
navigator.nowAt(HomePanelsScreen)
navigator.performAction(Action.CloseURLBarOpen)
navigator.nowAt(NewTabScreen)
```

And also, same lines, after `navigator.performAction(Action.OpenNewTabFromTabTray)`